examples/create-react-app/src/components/period-selector-dialog.js

```diff
@@ -4,7 +4,7 @@ import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 import RaisedButton from 'material-ui/RaisedButton';
 import Snackbar from 'material-ui/Snackbar';
 import PropTypes from 'prop-types';
-import PeriodSelectorDialog from '@dhis2/d2-ui-period-selector-dialog';
+import { PeriodSelectorDialog } from '@dhis2/d2-ui-dialogs';
```

Comment:
Does importing with this syntax mean that we get all the dialogs in our app bundle?

Reply:
No, it's still shaken out.